src/intentManager.js

```diff
@@ -108,8 +116,8 @@ define([
         writeInnerTagXML(xmlWriter, 'extra', properties.androidIntentExtra);
         if (properties.docTemplate) {
             writeInnerTagXML(xmlWriter, 'extra', {
-                'cc:print_template_reference': properties.docTemplate
-            });
+                'cc:print_template_reference': '.'
```

Comment:
It does currently works with the printer-integration branch, so this PR should be good to go as long as we remember to come back and resolve this issue.